proto/autokitteh/user_code/v1/handler_svc.proto

```diff
@@ -167,9 +167,24 @@ message HandlerHealthResponse {
   string error = 1;
 }
 
+message ExecuteReplyRequest {
+  string runner_id = 1;
+  autokitteh.values.v1.Value result = 2;
+  string error = 3;
+  repeated autokitteh.user_code.v1.Frame traceback = 4;
+}
+
+message ExecuteReplyResponse {
+  string error = 1;
```

Comment:
ProgramError?

Reply:
Other PR, don't want too many variables in this one.